it/connectors/propertyLinking/PropertyLinkConnectorISpec.scala

```diff
@@ -60,4 +60,33 @@ class PropertyLinkConnectorISpec extends ISpecBase {
       await(connector.getMyOrganisationAgents()) shouldBe testAgentList
     }
   }
+  
+  "agentAppointmentChange (POST)" should {
+    "return a successful AgentAppointmentChangesResponse" in {
+      implicit val testHeaderCarrier: HeaderCarrier = HeaderCarrier()
+
+      val agentChangeResponse = AgentAppointmentChangesResponse(appointmentChangeId = "1")
+
+      val agentChangeRequest = AgentAppointmentChangeRequest(
+        agentRepresentativeCode = 123456,
+        scope = "PROPERTY_LIST",
+        action = "APPOINT",
+        propertyLinkIds = Some(List("123L")),
+        listYears = Some(List("2023")))
+
+
+      val jsonResponse = Json.obj("appointmentChangeId" -> "1")
+
+      stubFor(
+        post("/property-linking/my-organisation/agent/submit-appointment-changes")
+          .willReturn(
+            aResponse
+              .withStatus(CREATED)
+              .withBody(jsonResponse.toString())
+          )
+      )
```

Comment:
Can we add a check on the stub to verify that the correct body has been passed please? There is an example of how to do this at https://github.com/hmrc/business-rates-check/blob/main/it/uk/gov/voa/businessrates/check/controllers/CheckBuilderControllerISpec.scala#L121

Reply:
Yep no worries, added this request body check in last commit.